app/controllers/associated_users_controller.rb

```diff
@@ -67,7 +67,7 @@ def edit
   end
 
   def create
-    creator = AssociatedUserCreator.new(project_role_params)
+    creator = AssociatedUserCreator.new(project_role_params, current_user)
```

Comment:
@Stuart-Johnson Looks like you have a bunch of extra changes. Maybe because v3.1.0 isn't up-to-date with v3.0.0b?

Reply:
Yeah, there was some git weirdness going on. It's resolved now.